src/live_restore/live_restore_fs.c

```diff
@@ -209,9 +209,26 @@ __live_restore_fs_find_layer(WT_FILE_SYSTEM *fs, WT_SESSION_IMPL *session, const
         return (0);
     }
 
+    /*
```

Comment:
I'll need to think a bit on this change. 
Stop files and the migration state are intended to hide the existence of source files from the user, while `find_layer` is an internal function. I can see the logic behind moving it - and all of our tests still work - but it feels weird to me that we'd hide state from internal functions.

Let's wait on @luke-pearson for a second opinion.

Reply:
Actually this change makes a lot of sense to me, stop files are meant to hide the source from both the user and us really. We should never touch the source in the event the migration is complete but the previous implementation would, additionally a stop file dictates that whatever is in the source is not relevant so again it should not be touched in this instance.

Reply:
Fair enough. In that case things LGTM